src/node_file.cc

```diff
@@ -116,6 +116,24 @@ inline int64_t GetOffset(Local<Value> value) {
   return IsSafeJsInt(value) ? value.As<Integer>()->Value() : -1;
 }
 
+inline int GetValidatedFd(Environment* env, Local<Value> value) {
+  if (!value->IsInt32()) {
+    env->isolate()->ThrowException(ERR_INVALID_ARG_TYPE(
```

Comment:
I think this should be a `ValidateInt32` if we want to start moving the validation to C++. Also this is not entirely on-par with the original implementation - the original implementation would print the value in a readable format in the case it's not a number, and this omits that. We could also just don't move the error validation code now and leave it in JS if it's not ready to take on implementing proper argument validation + printing in C++.

Reply:
@joyeecheung Thank you for your comment.
You are right. As you said, I think we need to think more about the implementation of `GetValidateFd()`. First, I will remove this addition to C++ and change it to the original implementation.